### PR TITLE
Shield Study 4 (Site Enhance) Static Pages

### PR DESCRIPTION
**Pull request by gregglind**
Consent Page for Shield Study 4 - Site Enhance.

See:  https://bugzilla.mozilla.org/show_bug.cgi?id=1292684 for tracking.

### CODE REVIEW

**Comment by gregglind** [comment]
@EnTeQuAk I think this even has the right tests and urls changes!

File: src/olympia/pages/templates/pages/shield_study_4.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (muffinresearch) [inline src/olympia/pages/templates/pages/shield_study_4.html]: Does this need to be XHTML?

**Comment by muffinresearch** [comment]
There's some python lint errors.

File: src/olympia/pages/templates/pages/shield_study_4.html (hunk 1 above)
Comment (gregglind) [inline src/olympia/pages/templates/pages/shield_study_4.html]: Nope.  Happy to pull that out.  What do you recommend instead?

**Comment by gregglind** [comment]
All fixed, please tell me this is going in today :)

**Comment by muffinresearch** [comment]
Is there a related issue number for this?

File: src/olympia/pages/templates/pages/shield_study_4.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title>Enhance Websites</title>
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_4/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Enhance Websites</h1>
+          <a href=https://addons.mozilla.org/firefox/downloads/latest/site-enhance-shield-study/?src=shield-study-4><button class="button primary large">Try Site Enhance</button></a>
```
Comment (muffinresearch) [inline src/olympia/pages/templates/pages/shield_study_4.html]: missing quotes

File: src/olympia/pages/templates/pages/shield_study_4.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          <!-- <button class="button primary">Enable Enhance Websites</button> -->
+        </div>
+        <div class="content"><p>Mozilla wants to know if having a Firefox experience tailored to the way you use the internet makes browsing the web more enjoyable. Participating in this study will help us find out!</p>
+<h2 id="what-will-happen-next">What will happen next</h2>
```
Comment (muffinresearch) [inline src/olympia/pages/templates/pages/shield_study_4.html]: Would be good to fix the indent to make this easier to read.

File: src/olympia/pages/templates/pages/shield_study_4.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+<li>Type <code>about:addons</code> into the location bar, and press enter.</li>
+<li>Find <code>Site Enhance Shield Study</code> in the <strong>extensions</strong> list pane.</li>
+<li>Click the <code>remove</code> button.</li>
+<li>If you opt out of the study early we will ask you to fill out a survey. We’re interested in hearing about your experience, even if you didn’t participate in the entire study.</li>
+</ol>
+<p>Opting out of a study does not prevent you from participating in future studies.</p>
+<h2 id="your-privacy">Your privacy</h2>
+<h3 id="all-shield-studies">All Shield Studies</h3>
+<p>Every Shield Study collects data about important study events, such as install, uninstall, daily usage, and end of study. Studies also include an optional survey.</p>
+<p>This data is associated with <a href="http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/index.html">Firefox Telemetry collection</a>.</p>
```
Comment (muffinresearch) [inline src/olympia/pages/templates/pages/shield_study_4.html]: Link can be https.

File: src/olympia/pages/templates/pages/shield_study_4.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    <meta charset="utf-8" />
+    <title>Enhance Websites</title>
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_4/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Enhance Websites</h1>
+          <a href=https://addons.mozilla.org/firefox/downloads/latest/site-enhance-shield-study/?src=shield-study-4><button class="button primary large">Try Site Enhance</button></a>
+          <!-- <button class="button primary">Enable Enhance Websites</button> -->
```
Comment (muffinresearch) [inline src/olympia/pages/templates/pages/shield_study_4.html]: Can this be deleted if it's not needed?

File: src/olympia/pages/templates/pages/shield_study_4.html (hunk 2 above)
Comment (muffinresearch) [inline src/olympia/pages/templates/pages/shield_study_4.html]: Why do you have the `button` inside the `a`? Can you not style the link instead?

Comment (gregglind) [inline src/olympia/pages/templates/pages/shield_study_4.html]: I just took what john gruen made in his template and did that :)

File: src/olympia/pages/templates/pages/shield_study_4.html (hunk 5 above)
Comment (gregglind) [inline src/olympia/pages/templates/pages/shield_study_4.html]: done

File: src/olympia/pages/templates/pages/shield_study_4.html (hunk 4 above)
Comment (gregglind) [inline src/olympia/pages/templates/pages/shield_study_4.html]: done

File: src/olympia/pages/templates/pages/shield_study_4.html (hunk 3 above)
Comment (gregglind) [inline src/olympia/pages/templates/pages/shield_study_4.html]: it's autogenerated from markdown, alas.

File: src/olympia/pages/templates/pages/shield_study_4.html (hunk 2 above)
Comment (gregglind) [inline src/olympia/pages/templates/pages/shield_study_4.html]: I don't know why John made that choice, and I am loathe to change it.

**Comment by gregglind** [comment]
Edited pull to ref bugzilla number.

**Comment by muffinresearch** [comment]
I'm going to land this and cherry pick it into a new tag and push it to stage. I'll update this issue when it's ready to review.

**Comment by muffinresearch** [comment]
It's in tag 2016.08.18-1 and that's on stage now.

**Comment by muffinresearch** [comment]
@gregglind please verify on stage https://addons.allizom.org/en-US/firefox/shield_study_4

**Comment by gregglind** [comment]
Thanks.  I see it works fine!

**Pull request merged**